content/en/docs/releases/next-release-preview/index.md

```diff
@@ -10,3 +10,26 @@ in the next release of Spinnaker. These notes will be prepended to the release
 changelog.
 
 ## Coming Soon in Release 1.36
+
+### Enhanced pipeline batch update feature
+
+Batch update operation is now atomic. Deserialization issues are addressed. 
+
+Configurable controls are added to decide whether cache should be refreshed while checking for duplicate pipelines:
+```yaml
+controller:
+   pipeline:
+      save:
+         refreshCacheOnDuplicatesCheck: false // default is true 
+```
+
+Batch update call now responds with a status of succeeded and failed pipelines info. The response will be a map containing information in the following format:
+
+``` 
+[
+    "successful_pipelines_count"  : <int>,
+    "successful_pipelines"        : <List<String>>,
```

Comment:
What are the elements of the list?

Reply:
It's a list of successful pipeline names. Added a sentence now.